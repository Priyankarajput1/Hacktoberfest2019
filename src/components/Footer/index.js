import PropTypes from 'prop-types';
import React from 'react';
import styled from '@emotion/styled';

import greenberryLogo from '../../images/greenberry_logo.svg';
import hacktoberfestLogo from '../../images/hacktoberfest_logo.svg';

const HacktoberButton = styled.a`
  height: 100%;
  display: block;
  justify-self: end;
  & > img {
    height: 100%;
    width: 100%;
  }
  transition: all 0.4s cubic-bezier(0.165, 0.84, 0.44, 1);
  &:hover {
    transform: scale(1.04, 1.04);
  }
`;

const GreenberryButton = styled.a`
  height: 100%;
  display: block;
  justify-self: start;
  & > img {
    height: 100%;
    width: 100%;
  }
  transition: all 0.4s cubic-bezier(0.165, 0.84, 0.44, 1);
  &:hover {
    transform: scale(1.04, 1.04);
  }
`;

const Wrapper = styled.footer`
  position: relative;
  height: 200px;
  width: 100%;
  grid-area: footer;

  display: grid;
  grid-template-columns: repeat(2, 50%);

  padding: 0 135px;
`;

const Footer = () => (
  <Wrapper>
    <GreenberryButton target="_blank" href="https://www.greenberry.nl/">
      <img src={greenberryLogo} alt="Greenberry" />
    </GreenberryButton>
    <HacktoberButton
      target="_blank"
      href="https://hacktoberfest.digitalocean.com/"
    >
      <img src={hacktoberfestLogo} alt="Hacktoberfest 2019" />
    </HacktoberButton>
  </Wrapper>
);

Footer.propTypes = {};

Footer.defaultProps = {};

export default Footer;
