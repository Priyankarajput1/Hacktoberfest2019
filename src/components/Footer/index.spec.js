import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './index';

describe('components/Footer', () => {
  it('should render', () => {
    const component = renderer.create(<Footer />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
