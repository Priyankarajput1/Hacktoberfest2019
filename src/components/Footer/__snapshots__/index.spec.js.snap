// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Footer should render 1`] = `
.emotion-4 {
  position: relative;
  height: 200px;
  width: 100%;
  grid-area: footer;
  display: grid;
  grid-template-columns: repeat(2,50%);
  padding: 0 135px;
}

.emotion-0 {
  height: 100%;
  display: block;
  justify-self: start;
  -webkit-transition: all 0.4s cubic-bezier(0.165,0.84,0.44,1);
  transition: all 0.4s cubic-bezier(0.165,0.84,0.44,1);
}

.emotion-0 > img {
  height: 100%;
  width: 100%;
}

.emotion-0:hover {
  -webkit-transform: scale(1.04,1.04);
  -ms-transform: scale(1.04,1.04);
  transform: scale(1.04,1.04);
}

.emotion-2 {
  height: 100%;
  display: block;
  justify-self: end;
  -webkit-transition: all 0.4s cubic-bezier(0.165,0.84,0.44,1);
  transition: all 0.4s cubic-bezier(0.165,0.84,0.44,1);
}

.emotion-2 > img {
  height: 100%;
  width: 100%;
}

.emotion-2:hover {
  -webkit-transform: scale(1.04,1.04);
  -ms-transform: scale(1.04,1.04);
  transform: scale(1.04,1.04);
}

<footer
  className="emotion-4 emotion-5"
>
  <a
    className="emotion-0 emotion-1"
    href="https://www.greenberry.nl/"
    target="_blank"
  >
    <img
      alt="Greenberry"
      src="totally real file"
    />
  </a>
  <a
    className="emotion-2 emotion-3"
    href="https://hacktoberfest.digitalocean.com/"
    target="_blank"
  >
    <img
      alt="Hacktoberfest 2019"
      src="totally real file"
    />
  </a>
</footer>
`;
